Learning_Guide/Coaching.md

```diff
@@ -0,0 +1,51 @@
+## Coaching Guide
+
+_Original author: Diana Vashti. Thanks Di!_
+
+As a coach, the feel of the week is a lot different than working on goals. You have no specs, no codebase, and instead of a team of 1 or 2, you are working with the whole guild. You may have back-to-back “coach-queue” appointments. With the change of pace and scenery comes a new set of skills that are ideal for sharpening during a week of coaching.
+
+## How do I know if I am coaching?
+
+You know you're coaching if the coaching coordinator tells you you're coaching. You'll be asked to vote for [goal 135](http://jsdev.learnersguild.org/goals/135-Coaching.html).
+
+Once you make it to level 3 you will become eligible to coach. If it’s your first week at level 3 you will likely coach that week.
+
+## What does coaching entail?
+
+Check out the [Technical Coach](../Game_Manual/Levels_and_Roles.md#technical-coach) outline in [Levels and Roles](../Game_Manual/Levels_and_Roles.md).
+
+### Okay, but what does my day actually look like?
+
+As a coach you have 3 main responsibilities:
+
+1. **Maintain available hours for coach-queue appointments**. These are half-hour blocks learners can schedule through the coach-queue app when they are feeling blocked. Your google calendar will be the source of your availability so make sure you block off time you do not have available for appointments there.
+1. **Join morning stand-ups**.
+1. **Escalating code blocks and issues to SEPs if you are unable to resolve the learners block during your appointment** (ideally, if after 15 minutes you do not feel you are on a solid path to debugging the issue, a coach should find an available SEP to bring in.)
+
+**Tip**: It is a good practice to block off time at the end and beginning of each day to focus on code reviews, and to keep as much middle-of-the-day time as possible available for coach-queue appointments.
```

Comment:
Is this tip necessary any longer?  This was relevant when coaches were assigned appointments for each request...